src/libcadet/Weno.hpp

```diff
@@ -57,7 +57,7 @@ class Weno
 	/**
 	 * @brief Creates the WENO scheme
 	 */
-	Weno() : _order(maxOrder()), _boundaryTreatment(BoundaryTreatment::ReduceOrder), _intermediateValues(3 * maxOrder() * sizeof(active)) { }
+	Weno() : _epsilon(1e-10), _order(maxOrder()), _boundaryTreatment(BoundaryTreatment::ReduceOrder), _intermediateValues(3 * maxOrder() * sizeof(active)) { }
```

Comment:
@sleweke are all these changes in WENO OK for merge into `master`?

Reply:
This has no influence, weno_eps is still required as input and overwrites the default 1e-10